### PR TITLE
Added function: Filter by winning percentage

### PR DESCRIPTION
**Pull request by dickstar**
Thanks for your sharing.
I added a function: "Filter by winning percentage", help me to focus on the high winning probability giveaways.
It only add some necessary code like your style, and didn't conflict with your original programs at all.
I wish you will also like this function.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Professional UI plugin for steamgifts.com website
 ### Hide blacklisted games 
 ### Hide giveaways with too high contributor value 
 ### Filter by giveway level 
+### Filter by winning Percent
```
Comment (bmxmiso) [inline README.md]: stick to the general format and write "Percent" using lowercase

File: manifest.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "default_locale": "en",
     "author": "MeDAN",
     "manifest_version": 2,
-    "version": "2.5.1",
+    "version": "2.5.2",
```
Comment (bmxmiso) [inline manifest.json]: New features increment MINOR version so please use "2.6.0" number.

**Review by bmxmiso — CHANGES_REQUESTED** [review]
Just small requests

**Comment by micso** [comment]
bmxmiso is my 2nd account (forgot to switch) ;)

File: README.md (hunk 1 above)
Comment (dickstar) [reply README.md]: of course, thank u

**Review by micso — APPROVED** [review]

**Comment by micso** [comment]
Published

**Pull request merged**